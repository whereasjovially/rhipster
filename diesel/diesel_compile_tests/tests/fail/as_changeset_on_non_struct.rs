#[macro_use]
extern crate diesel;

table! {
    users {
        id -> Integer,
        name -> Text,
        hair_color -> Nullable<Text>,
    }
}

#[derive(AsChangeset)]
#[table_name = "users"]
enum User {}

fn main() {}
