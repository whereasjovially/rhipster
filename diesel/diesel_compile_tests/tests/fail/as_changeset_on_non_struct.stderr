error: This derive can only be used on structs
  --> $DIR/as_changeset_on_non_struct.rs:12:10
   |
12 | #[derive(AsChangeset)]
   |          ^^^^^^^^^^^
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
